src/components/MenuItems.js

```diff
@@ -46,7 +46,7 @@ export default function MenuItems() {
       )}
       {admin && (
       <NavLink
-        to="/login"
+        to="/dest"
         style={({ isActive }) => (isActive ? activeStyle : undefined)}
       >
         ADD DESTINATION
```

Comment:
Hi @danmainah , 


I think these changes are already done in the dev branch

![image](https://user-images.githubusercontent.com/63932912/143073384-0842a12d-87ac-40c4-acd6-2e03d77ef8a6.png)
![image](https://user-images.githubusercontent.com/63932912/143073490-8abff44d-44b2-4508-919d-cf21b3b2bc6f.png)


- All links on Navbar now are working fine. Make sure you pull the last updates from the dev branch.

Reply:
okay let me  recheck again